pkg/kn/flags/podspec.go

```diff
@@ -156,7 +156,8 @@ func (p *PodSpecFlags) AddFlags(flagset *pflag.FlagSet) []string {
 		"Add a volume from a ConfigMap (prefix cm: or config-map:) a Secret (prefix secret: or sc:), "+
 			"an EmptyDir (prefix ed: or emptyDir:) or a PersistentVolumeClaim (prefix pvc: or persistentVolumeClaim). "+
 			"Example: --volume myvolume=cm:myconfigmap, --volume myvolume=secret:mysecret or --volume emptyDir:myvol:size=1Gi,type=Memory. "+
-			"You can use this flag multiple times. "+
+			"You can use this flag multiple times. You can also specify if a volume is readOnly or not. "+
+			"Example: --volume myvolume=cm:myconfigmap:readOnly=True or --volume emptyDir:myvol:size=1Gi,type=Memory,readOnly=false. "+
```

Comment:
```suggestion
			"Example: --volume myvolume=cm:myconfigmap:readOnly=true"+
```

Reply:
Maybe let's rephrase the whole help message like in 

> Add a volume. This option can be given multiple times and has the format: <name>=<prefix:objName:options>. "name" is the name of the volume to create, prefix can be "cm" or "config-map" for a ConfigMap volume, "secret" or "sc" for a Secret volume, "ed" or "emptyDir" for an emptyDir volume or "pvc:" or "persistentVolumeClaim" for a persistent volume. objName is the referenced object to use as a volume (e.g. the name of the ConfigMap). options is a comma-separated key-value list for additional options specific to the volume. Use "readOnly=true" as an option to specify the writable volumes being read-only. Example: "--volume myVolume=pvc:my-storage-claim:readOnly=true"

Reply:
tbh, I don't think you can specify `readOnly` to `--volume` but only to `--mount`. Is this correct?

Reply:
I think there's a specual case for `PersistenVolumeClaimVolume` that it can have `readOnly` that will enforce it on mount afterwards.

Reply:
However, yes that should be probably `--mount` flag option.